packages/react-vapor/src/components/multiValueInputs/MultiValuesInput.tsx

```diff
@@ -56,6 +69,10 @@ export const MultiValuesInput: React.FunctionComponent<MultiValuesInputProps> =
                                 parentProps.removeBox(cData.id);
                             }
                         }}
+                        placeholder={index >= limit ? reachedLimitPlaceholder : placeholder}
+                        disabled={index >= limit}
+                        innerInputClasses={index >= limit && 'mod-no-border py1 input-wider-text-box'}
+                        disabledTooltip={index >= limit ? "this input can't be edited." : undefined}
```

Comment:
`"This input can't be edited."` Should we confirm this string with someone from doc team, or allow passing an optional prop for `disabledTooltip`?

Reply:
Good point, I think it would be interesting to allows this feature to be customizable  .